modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc

```diff
@@ -393,15 +449,15 @@ include::example$TransactionsExample.java[tag=full-logging,indent=0]
 
 
 == Tracing
+[.status]#Developer Preview#
 
-The 1.1.6 release introduces support for OpenTelemetry and OpenTracing.
-
-If the underlying Couchbase Java SDK is configured for OpenTelemetry or OpenTracing, then transaction spans will now be output automatically.
+The 1.1.6 release introduces support, at a Developer Preview level, for detailed tracing.
+This telemetry is particularly useful for monitoring performance.
 
-See the xref:howtos:observability-tracing.adoc[Couchbase Java SDK Request Tracing documentation] for more guidance on configuring the Couchbase Java SDK appropriately.
+Note: Developer Preview here means the feature works and is safe to use, but the naming of spans and attributes, and which spans are created, may change before this feature is generally available.
```

Comment:
you could refer to the interface taxonomy as uncommitted.

Reply:
Ah well it's deliberately not as the only API is parentSpan(RequestSpan span) in PerTransactionConfigBuilder and this isn't going to change - RequestSpan is nailed down.